app/assets/javascripts/index/note.js

```diff
@@ -52,20 +52,17 @@ OSM.Note = function (map) {
           OSM.loadSidebarContent(path, function () {
             initialize(path, id, moveToNote);
           });
+        },
+        error: function (xhr) {
+          $(form).find("#comment-error").text(xhr.responseText);
+          $(form).find("#comment-error").prop("hidden", false);
```

Comment:
I was copying the calls from changeset.js where they are not chained:

https://github.com/openstreetmap/openstreetmap-website/blob/98f8bbace3bec93a23fd9924f0a02c9d41c9cf69/app/assets/javascripts/index/changeset.js#L50-L51

Reply:
> Also should the success callback hide the error again

It doesn't have to because it rewrites the entire sidebar contents.

Added test to check that:
https://github.com/openstreetmap/openstreetmap-website/pull/4462/files#diff-960e6e9200fa7ea5eed049f5de24d807cf7c7a4c0434875d23139ff261ddc0feR73-R79

Reply:
Changed `$(form).find("#comment-error")` to chained calls.